pageserver/src/page_service.rs

```diff
@@ -772,8 +775,9 @@ impl PageServerHandler {
         pgb.write_message_noflush(&BeMessage::CopyOutResponse)?;
         pgb.flush().await?;
 
-        // Send a tarball of the latest layer on the timeline
-        {
+        // Send a tarball of the latest layer on the timeline. Compress if not
+        // fullbackup. TODO Compress in that case too (tests need to be updated)
+        if full_backup {
```

Comment:
Why do we need to avoid compression for full backup - it is not clear from the comment...

Reply:
Not avoiding it. There's a TODO saying I want to do it. But it will require a lot of changes and it's unnecessary now. Should I clarify the comment somehow?